sql/firefox_kpi_dashboard_v1.sql

```diff
@@ -0,0 +1,231 @@
+    var mean = total / n;
+    var resamples = jackknife_resamples(counts_per_bucket);
+    var jk_means = resamples.map(x => array_avg(x));
+    var mean_errs = jk_means.map(x => Math.pow(x - mean, 2));
+    var bucket_std_err = Math.sqrt((n-1) * array_avg(mean_errs));
+    var std_err = n * bucket_std_err;
+    var z_score = Math.sqrt(2.0) * erfinv(0.90);
+    var hi = total + z_score * std_err;
+    var lo = total - z_score * std_err;
+    return {
+        "total": total,
+        "low": lo,
+        "high": hi,
+        "pm": hi - total,
+        "nbuckets": n,
+    };
+}
+
+return sum_buckets_with_ci(a);
+""";
+  --
+WITH
+  desktop_base AS (
+  SELECT
+    *
+  FROM
+    `moz-fx-data-derived-datasets.telemetry.firefox_desktop_exact_mau28_by_dimensions_v1` ),
+  --
+  nondesktop_base AS (
+  SELECT
+    *
+  FROM
+    `moz-fx-data-derived-datasets.telemetry.firefox_nondesktop_exact_mau28_by_dimensions_v1` ),
+  --
+  fxa_base AS (
+  SELECT
+    *
+  FROM
+    `moz-fx-data-derived-datasets.telemetry.firefox_accounts_exact_mau28_by_dimensions_v1` ),
+  --
+  per_bucket AS (
+  SELECT
+    'desktop_global' AS datasource,
+    'actual' AS type,
+    submission_date,
+    id_bucket,
+    SUM(mau) AS mau
+  FROM
+    desktop_base
+  GROUP BY
+    id_bucket,
+    submission_date
+  UNION ALL
+  SELECT
+    'desktop_tier1' AS datasource,
+    'actual' AS type,
+    submission_date AS date,
+    id_bucket,
+    SUM(IF(country IN ('US',
+          'FR',
+          'DE',
+          'GB',
+          'CA'),
+        mau,
+        0)) AS mau
+  FROM
+    desktop_base
+  GROUP BY
+    id_bucket,
+    submission_date
+    --
+  UNION ALL
+    --
+  SELECT
+    'nondesktop_global' AS datasource,
+    'actual' AS type,
+    submission_date,
+    id_bucket,
+    SUM(mau) AS mau
+  FROM
+    nondesktop_base
+  GROUP BY
+    id_bucket,
+    submission_date
+  UNION ALL
+  SELECT
+    'nondesktop_tier1' AS datasource,
+    'actual' AS type,
+    submission_date AS date,
+    id_bucket,
+    SUM(IF(country IN ('US',
+          'FR',
+          'DE',
+          'GB',
+          'CA'),
+        mau,
+        0)) AS mau
+  FROM
+    nondesktop_base
+  GROUP BY
+    id_bucket,
+    submission_date
+    --
+  UNION ALL
+    --
+  SELECT
+    'fxa_global' AS datasource,
+    'actual' AS type,
+    submission_date,
+    id_bucket,
+    SUM(mau) AS mau
+  FROM
+    fxa_base
+  GROUP BY
+    id_bucket,
+    submission_date
+  UNION ALL
+  SELECT
+    'fxa_tier1' AS datasource,
+    'actual' AS type,
+    submission_date AS date,
+    id_bucket,
+    SUM(seen_in_tier1_country_mau) AS mau
+  FROM
+    fxa_base
+  GROUP BY
+    id_bucket,
+    submission_date ),
+  --
+  with_ci AS (
+  SELECT
+    datasource,
+    type,
+    submission_date,
+    sum_buckets_with_ci(ARRAY_AGG(mau)).*
+  FROM
+    per_bucket
+  GROUP BY
+    datasource,
+    type,
+    submission_date ),
+  --
+  with_forecast AS (
+  SELECT
+    datasource,
+    type,
+    submission_date AS date,
```

Comment:
I think we should quote identifiers that are keywords, even if they aren't [reserved keywords](https://cloud.google.com/bigquery/docs/reference/standard-sql/lexical#reserved-keywords)
```suggestion
    submission_date AS `date`,
```